index.bs

```diff
@@ -17,11 +17,14 @@ Markup Shorthands: css no, markdown yes
 <pre class=anchors>
 spec: storage; urlPrefix: https://storage.spec.whatwg.org/
     type: dfn
+        text: bottle map; url: bottle-map
         text: bucket map; url: bucket-map
         text: mode; url: bucket-mode
         text: obtain a local storage shelf; url: obtain-a-local-storage-shelf
         text: storage bucket; url: storage-bucket
+        text: storage bottle; url: storage-bottle
```

Comment:
super nit: if this is alphabetical, this should go above storage bucket

Reply:
done